src/tests/resistance/handshake.rs

```diff
@@ -38,3 +42,42 @@ async fn r001_HANDSHAKE_reject_if_user_agent_too_long() {
     synth_node2.shut_down().await;
     node.stop().unwrap();
 }
+
+#[allow(non_snake_case)]
+#[tokio::test]
+async fn r002_HANDSHAKE_reject_if_server_too_long() {
```

Comment:
Maybe these two tests could be placed under the same SPEC test R001 like:
r001_t1_HANDSHAKE_xxx
r001_t2_HANDSHAKE_xxx

So we have different variants of the test: "test with wrong handshake fields"

Reply:
Because even the SPEC texts are pretty similar, they both revolve around doing an invalid handshake thing.

Reply:
Done